subsys/bluetooth/mesh/lightness_srv.c

```diff
@@ -175,6 +166,15 @@ static void handle_linear_get(struct bt_mesh_model *model,
 	handle_light_get(model, ctx, buf, LINEAR);
 }
 
+void lightness_srv_disable_control(struct bt_mesh_lightness_srv *srv)
```

Comment:
Could it be `static inline` in `lightness_internal.h`?

Reply:
It could, but I don't see enormous advantages in doing that?